src/api-wrappers/AXUIElement.swift

```diff
@@ -200,6 +205,15 @@ extension AXUIElement {
         return title != "" && Applications.isAndroidEmulator(runningApp)
     }
 
+    private static func colorSlurp(_ runningApp: NSRunningApplication) -> Bool {
+        // ColorSlurp presents its dialog as a kAXSystemDialogSubrole, so we need a special check
+        return runningApp.bundleIdentifier == "com.IdeaPunch.ColorSlurp"
+    }
+
+    private static func crossoverWindow(_ runningApp: NSRunningApplication, _ role: String?, _ subrole: String?, _ level: CGWindowLevel) -> Bool {
+        runningApp.bundleIdentifier == nil && role == kAXWindowRole && subrole == kAXUnknownSubrole && level == 0
```

Comment:
This check here is too broad. Many app could have `runningApp.bundleIdentifier == nil`. We don't want those to have all their windows shown without filtering.

Maybe try and pin-point this CrossOver app with something like what I did for `androidEmulator`?

As it stands, I can't merge this, unfortunately. It's too broad.

Reply:
It looks like anything launched from Game Porting Toolkit/CrossOver/WINE has a "localized" app name of `wine64-preloader`. I added that to the condition.

Reply:
Actually this won't work. Looking into it more...